readme.markdown

```diff
@@ -217,3 +217,30 @@ We have received reports that the `idris-stay-in-current-window-on-compiler-erro
     (dolist (x '("*idris-notes*" "*idris-holes*" "*idris-info*"))
       (plist-put (cdr (assoc x popwin:special-display-config)) :noselect t))))
 ```
+
+
+### [`frames-only-mode`](https://github.com/davidshepherd7/frames-only-mode) and excessive buffer pop-ups
```

Comment:
I don't really like the term "excessive" here --- many people, myself included, prefer a workflow with many small, short-lived buffers. How about "and many buffer pop-ups"?